camino-messenger-bot-supplier.yaml.example

```diff
@@ -16,7 +18,7 @@ developer_mode: true
 matrix_key: YOUR_PRIVATE_KEY_HEX
 
 # Hostname of the Matrix Server
-matrix_host: matrix.camino.network
+matrix_host: dev.matrix.camino.network
```

Comment:
I think its outdated

Reply:
I put 
`dev.matrix.camino.network` in examples
and `messenger.chain4travel.com` in docker-compose.yml

Is this ok?